src/main/scala/it/unibo/authsim/library/dsl/otp/util/OTPHelpers.scala

```diff
@@ -0,0 +1,72 @@
+package it.unibo.authsim.library.dsl.otp.util
+
+import it.unibo.authsim.library.dsl.HashFunction
+import it.unibo.authsim.library.dsl.policy.model.StringPolicies.OTPPolicy
+
+import javax.crypto.Mac
+import javax.crypto.spec.SecretKeySpec
+import scala.collection.immutable.NumericRange
+import scala.language.postfixOps
+import scala.util.Random
+
+/**
+ * ''OTPHelpers'' is object that has the usefull functions for otp implementation.
+ */
+object OTPHelpers:
+
+  implicit class RichInt(base: Int):
+    /**
+     * @return the next number of ''base''
+     */
+    def ++ : Int = base + 1
```

Comment:
For me, this is a bad implicit: it defines an operation with a specific name, which carries a specific semantic (return the value of `base` and then increment the value of `base` by 1), and changes its meaning, which is a lot confusing.

The two suggested changes are:
* Remove the implicit and directly update the variables where this implicit is used.
* Change the name of the method in something like `next`, to move away from the meaning of `++`.

Reply:
Ok, I remove it because I only use it once.